client/src/main/java/io/pravega/client/stream/notifications/Observable.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream.notifications;
+
+import java.util.concurrent.ScheduledExecutorService;
+
+/**
+ * This represents an observable notification.
+ * @param <T>
+ */
+public interface Observable<T> {
```

Comment:
nit: can we name the methods "registerListener" and "unregisterListener"?

Reply:
Done.